blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -54,8 +54,12 @@ export class PipelinePage extends Component {
         const { location = {} } = this.context;
 
         const { organization, name, fullName, fullDisplayName } = pipeline || {};
-        const orgUrl = buildOrganizationUrl(organization);
-        const activityUrl = buildPipelineUrl(organization, fullName, 'activity');
+        
+        const organizationName = organization || AppConfig.getOrganizationName();
+        const organizationDisplayName = organization === AppConfig.getOrganizationName() ? AppConfig.getOrganizationDisplayName() : organization;
```

Comment:
@alvarolobato when will `organization === AppConfig.getOrganizationName()` be false? According to code, if it's false you are using `organization` and not organizationDisplayName? Maybe an example will be helpful.